test/Core.Test/AdminConsole/Services/RabbitMqIntegrationListenerServiceTests.cs

```diff
@@ -18,19 +19,24 @@ public class RabbitMqIntegrationListenerServiceTests
     private const string _queueName = "test_queue";
     private const string _retryQueueName = "test_queue_retry";
     private const string _routingKey = "test_routing_key";
+    private readonly DateTime _now = new DateTime(2014, 3, 2, 1, 0, 0, DateTimeKind.Utc);
     private readonly IIntegrationHandler _handler = Substitute.For<IIntegrationHandler>();
     private readonly IRabbitMqService _rabbitMqService = Substitute.For<IRabbitMqService>();
 
     private SutProvider<RabbitMqIntegrationListenerService> GetSutProvider()
     {
-        return new SutProvider<RabbitMqIntegrationListenerService>()
+        var sutProvider = new SutProvider<RabbitMqIntegrationListenerService>()
             .SetDependency(_handler)
             .SetDependency(_rabbitMqService)
```

Comment:
No change required here, but for future reference: `SutProvider` will automatically populate with NSubstitute mocks when you call `Create()` for any dependencies not explicitly set. You can retrieve the mocks using `GetDependency<T>` in order to arrange them. So you really only need to call `SetDependency` if you need something more specific than that.

That will also handle reinitializing the mocks for each test. As currently written I'm not certain that your mocks are reset at the start of each test.